ui/app/AppLayouts/Profile/views/PrivacyView.qml

```diff
@@ -166,10 +166,15 @@ Item {
                 StatusQControls.StatusSwitch {
                     id: switch1
                     checked: !localAccountSensitiveSettings.onlyShowContactsProfilePics
+                    onCheckedChanged: {
+                        if (localAccountSensitiveSettings.onlyShowContactsProfilePics !== !checked) {
```

Comment:
Does simply doing
onCheckedChanged: {
  localAccountSensitiveSettings.onlyShowContactsProfilePics = !checked
}
Have any issues?

Reply:
Yes this could potentially cause an unnecessary save operation in the settings :) I just avoid it. In fact, the best way for it - make some bidirectional binding api for StatusSwitch. But `!` usage in some place do not allow make it "nice".

Reply:
I think the first time the component is created this may happen, but after that it only triggered when checked is changed which means local settings needs to be updated. But I get it :)